plutus-core/plutus-core/src/PlutusCore/Default/Universe.hs

```diff
@@ -323,15 +314,12 @@ internal one -- it's a normal evaluation failure, but unlifting errors have this
 being "internal".
 -}
 
-instance KnownTypeAst DefaultUni Int64 where
-    toTypeAst _ = toTypeAst $ Proxy @Integer
-
 -- See Note [Integral types as Integer].
-instance HasConstantIn DefaultUni term => MakeKnownIn DefaultUni term Int64 where
+instance HasConstantIn DefaultUni term => MakeKnownIn DefaultUni term Int where
     makeKnown = makeKnown . toInteger
     {-# INLINE makeKnown #-}
 
-instance HasConstantIn DefaultUni term => ReadKnownIn DefaultUni term Int64 where
```

Comment:
huh, did we not need the `Int64` thing? I'm confused why we could get rid of this

Reply:
We don't use it atm, all our builtins use Int. It was added during int-cast to go through Int64 and fail afterwards